src/ResXManager/MainViewModel.cs

```diff
@@ -44,7 +44,18 @@ public MainViewModel(ResourceManager resourceManager, IConfiguration configurati
 
             try
             {
-                var folder = Settings.Default.StartupFolder;
+                var settings = Settings.Default;

```

Comment:
That's for sure a lot of code duplication here...

Reply:
Yeah it was quick and dirty. Fixed in latest.